### PR TITLE
Small Sputnik V updates

### PR DESCRIPTION
**Pull request by RLordan**
Minor updates to the sputnik v section while prepping vaccine distribution info. 

### Suggested reviewers (optional)

@rando2 @ajlee21 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -234,8 +234,17 @@ Of the 707 vaccinated healthcare workers, 71.3% of the 96.6% of respondents repo
...
+Participants will first receive an intramuscular dose of AZD1222 on day 1, followed by a dose of rAd26 on day 29. 
+Participants will be monitored from day 1 for 180 days in total. 
+The primary outcomes measured will include incidence of serious adverse events post first dose until the end of the study. 
+Secondary outcome measures will include incidence of local and systemic adverse events 7 days post each dose, a time course of antibody responses for the Spike protein and the presence of anti-SARS-CoV-2 neutralizing antibodies [@clinicaltrials:NCT04686773].
+
 Overall, there is hesitancy surrounding the management of the Sputnik V vaccine approval process and concerns over whether the efficacy data may be inflated due to a lack of asymptomatic testing within the trial.
-However, further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 
+However, the results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen.
```
Comment (RLordan) [inline content/22.vaccines.md:247]:
```suggestion
However, the interim results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 
```

Comment (RLordan) [inline content/22.vaccines.md]:
```suggestion
However, the interim results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 
```

**Review by ajlee21 — APPROVED** [review]
Looks good to me. Really interesting to hear about the combination. I'm curious what they're combining since they're both viral vectors, though i'm not sure what subunit they're both using and maybe that's what is being combined - just me thinking out loud

**Comment by RLordan** [comment]
> Looks good to me. Really interesting to hear about the combination. I'm curious what they're combining since they're both viral vectors, though i'm not sure what subunit they're both using and maybe that's what is being combined - just me thinking out loud

Thanks @ajlee21 It’s super interesting. So the Russian shot is strange because they use two different vectors, not combined, but administered in separate vaccines 28 days apart- Ad5 and ad26. This trial will replace the use of ad5 with the AZ vector. The idea being that these two vectors are less prevalent than ad5 in human populations (I think)

File: content/22.vaccines.md [hunk 2]
```diff
@@ -234,8 +234,17 @@ Of the 707 vaccinated healthcare workers, 71.3% of the 96.6% of respondents repo
 Of these individuals, 68% experienced joint and muscle pain, 54% had injection site pain, 11% reported redness and swelling, 40% had a fever, and 5% reported diarrhea. 
 Only 5% of the vaccinated participants experienced serious adverse events that required medical attention, of which one was monitored as an inpatient.
 
+Additionally, an Independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available [https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA].
```
Comment (rando2) [inline content/22.vaccines.md:237]:
```suggestion
Additionally, an Independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available [@url:https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA]. 
```

File: content/22.vaccines.md [hunk 3]
```diff
@@ -234,8 +234,17 @@ Of the 707 vaccinated healthcare workers, 71.3% of the 96.6% of respondents repo
 Of these individuals, 68% experienced joint and muscle pain, 54% had injection site pain, 11% reported redness and swelling, 40% had a fever, and 5% reported diarrhea. 
 Only 5% of the vaccinated participants experienced serious adverse events that required medical attention, of which one was monitored as an inpatient.
 
+Additionally, an Independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available [https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA]. 
+On December 21st, 2020, Gamaleya, AstraZeneca, R-Pharm, and the Russian Direct Investment Fund agreed to assess the safety and immunogenicity of the combined use of components of the AstraZeneca and University of Oxford AZD1222 (ChAdOx1) vaccine and the rAd26-S component of the Sputnik V vaccine in clinical trials [@url:https://rdif.ru/Eng_fullNews/6220/].
+This agreement hopes to establish scientific and business relations between the entities with an aim to co-developing a vaccine with long-term immunization.
```
Comment (rando2) [inline content/22.vaccines.md:239]:
```suggestion
This agreement hopes to establish scientific and business relations between the entities with an aim to co-develop a vaccine providing long-term immunization. 
```

**Review by rando2 — APPROVED** [review]
This looks good to me!

**Pull request merged**